### ISSUE TITLE
Polishing review form

### ISSUE THREAD

**Issue by mrshirts** [livecomsjournal/journal_information#66]
Now that we have submissions coming in, we need to make sure the review form reflects what we want to have reviewers say.

You can see the current form if you log in to editor accounts (I think? Maybe only I can see, let me know) and look at 
https://app.scholasticahq.com/journals/livecoms/settings?journal_settings_section=reviewer_form

**Questions we can't change:**

*Overall Rating*
Reviewer will rank from 1-5 stars.

*For this manuscript I recommend...*
Reviewer will choose between Accept, Revise and resubmit, and Reject

*Comments to editor*
Reviewer will be provided a blank text area to fill in.

We can add/change yes/no questions, and open response questions.  All added questions will be required. The current default questions are (only editor sees the answers unless indicated otherwise).  

 - "The paper makes a significant contribution to scholarship" (yes/no)
 - "The paper adequately engages with the relevant scholarly literature" (yes/no)
 - "The paper is professionally written, easy to read, and free from grammatical errors" (yes/no)
 - "The literature review was thorough given the objectives and content of the article" (yes/no)
 - "Comments to the author" (open response, can be seen by author)
 - "Is the content of this article appropriate for this journal (open response) 

I will update this issue with different questions I propose asking during the morning. Please add any suggestions.

**Comment by mrshirts** [comment]
I think I'd like to have all review questions explicitly listed in the author information section, for full transparency.  I'll START by taking questions from that part of website, turn them into questions here, THEN put them there in the website.

**Comment by mrshirts** [comment]
I think we will essentially have separate explicit questions for each article family, as we have startd laying out on the webpage.

**Comment by mrshirts** [comment]
Suggested overall questions (from brainstorming and looking at what we have on github.io.

I'm thinking of going to solely open response questions.  We won't be sorting through hundreds of submissions, so we can take the time to read through answers. Suggested open response questions. All except those noted will be seen by the author.

 - In what ways should the paper be improved to be easy to read, free from grammatical errors, have a professional presentation, and meet the article formatting guidelines laid out in the [author policies](https://livecomsjournal.github.io/authors/policies/)?
 - To what extent does the paper engage with current understanding in the scholarly community? If this is a revision, to what extent do the authors engage with the community participating on their GitHub version?
 
For the per/document questions: I don't think we can have a different reviewer form for different types of manuscripts.  So we can 1) explicitly put this text in the question, or 2) give the reviewers a link to a place on the GitHub.io site that lists the specific questions to answer for each type of document.
- For Best Practice Documents:
   - Would following the provided checklist help users avoid significant potential errors in simulation? Will the errors be profound and/or frequent?
   - Are all assertions well-sourced in published data (which may, in some cases, include data created for the paper)?
   - Are the explanations clear enough for researchers with only moderate training in simulation?
 - For Tutorials
   - Are files and necessary executables required to run the tutorial posted online in a permanent (or nearly so) way?
   - Are the tutorials suitable for the intended audience (e.g., researchers with only basic -- 
   - knowledge or advanced researchers)?
   - Do they include enough information on how to generalize the approach to handle other cases, and highlight major issues to consider?
   - Do the tutorials use commonly agreed on best practices?
   - Do the tutorial authors clearly provide, where possible, information (e.g., primary literature citations) about why particular choices are being made during the step-by-step process?
   - Does the tutorial require use of proprietary software or data? If so, are workarounds made available for reserachers who may not have access to such software or data?
 - For Lessons Learned
   - Is the motivation for the approach clearly described and reasonable given existing literature?
   - Are the “lessons” provided likely to be of value to a significant number of researchers, based on demonstrated broad applicability?
   - Is the procedure used fully described (and derived mathematically if appropriate) to enable a reader to reproduce the results?
    - Are any needed supported files provided?
    - Has data analysis been performed in conformance with best practices?
 - For Comparisons of Molecular Simulation Software
   - Does the manuscript include developers or advanced users of all programs being compared to ensure that comparisons are fair?
   - Are best practices being used in the simulation comparisons?
    - Are the methods applied to systems which for which reliable independent data are available? Are the systems considered of sufficient complexity that readers will find the results convincing and generalizable?
   - Have the authors addressed whether a naive user could expect similar performance and why or why not?
   - Have the authors clearly identified both (a) the version(s) of the molecular simulations programs used in the comparison (e.g., through revision control tags, a standard version numbering system, or equivalent) and (b) any modifications made to the source code to facilitate the comparison?
   - Do the authors supply run scripts, other supporting files, and analysis routines so that the comparison study can, in principle, be replicated? Does the article identify the version of the run scripts, etc.?
 - For Perpetual Reviews
   - Do the authors properly include information from the recent literature, including last 6 to 12 months?
   - Is this a good summary of its area which will be valuable to the field?
   - Do the authors include data and viewpoints that are contradictory to their own, if these exist?
   - (If a revision) Have the authors removed outdated material and qualified information that has been disproven or contradicted? Does the revision add the appropriate and relevant new material or is it becoming outdated?

- Any additional comments to the authors 
- Any additional comments to the editors (will NOT be seen by the author)

**Comment by dmzuckerman** [comment]
For the overall questions, I think significance should be discussed.  If you don't like yes/no, we could say, "Please comment on the level of significance of this work and its suitability for the journal.  Is it likely to have a strong positive impact on a set of readers?"

Also, I see a couple of typos:

For tutorials, this is spread over two bullets, though it's one item: Are the tutorials suitable for the intended audience (e.g., researchers with only basic --
knowledge or advanced researchers)?

For 'Lessons Learned', omit []: Are any needed support[ed] files provided?

**Comment by davidlmobley** [comment]
@mrshirts - I'm generally in favor of moving mostly towards free response questions, though I don't think one or two yes/no or straightforward questions are necessarily bad.

Most of your questions are good. But you aren't going to want to have them all in the review form because that would make it too long/too many boxes; I'd deal with just by linking to the review criteria online and asking them to assess how well it meets the (linked to) criteria for the specific category of document.

I would also want to point out (again) that they are encouraged to use the GitHub issue tracker to raise any issues they would like to discuss openly with the authors, and reserve the review form for issues which require more confidentiality. 

Another typo:

> In what general ways does the paper need to improve to be easy to read

"improved"

**Comment by dmzuckerman** [comment]
I think @davidlmobley makes an important point, implicitly: I think a lot of people write a review and then cut-and-paste into the review form.  This is what I do, anyway and I think we should make it easy for our reviewers to do so.  From this perspective, the yes/no questions are good.

I'm not sure about another point DM makes - namely that reviewers should feel obligated to comment on github.  Not only is this is another layer of work, but authors won't have a single list of what's needed to make their manuscript acceptable, which will be unpleasant.  Further, the reviewer may compromise anonymity if s/he refers to github comments.  

If a reviewer wants to make extensive comments perhaps with a thought toward becoming an author, then it would be good to use github ... but then it starts to smell like a conflict of interest.  My instinct is that we shouldn't encourage this.

**Comment by davidlmobley** [comment]
I'm not trying to say they should be obligated to comment on GitHub, actually. I'm just trying to say that they should feel free to if they like.

My thought was just that in some cases, raising points on GitHub will be a way to have a "discussion" rather than just make a set of demands -- especially on repositories where there is already (or starting to be) community involvement. e.g. I could imagine that in your error analysis paper, eventually for a subsequent version there might be considerable discussion on GitHub as to whether things should be done via "novel method X" or "novel method Y" and the reviewer might want to cross-reference a GitHub issue rather than just providing all feedback via the review form.

TL;DR: Not requiring GitHub, just giving it as an option.

**Comment by davidlmobley** [comment]
Specifically, we have this text in our reviewer instructions and we might just want to REMIND reviewers of it:

> One potentially unique aspect of the LiveCoMS review process is the opportunity to engage with authors publicly rather than anonymously if you so desire. Specifically, if you (as a reviewer) are interested in participating public discussion of the work and helping to improve it, you are free to comment on the issue tracker connected to the GitHub repository associated with the article, which can be a way to work together with the community to help authors improve their article. If you choose to do this, you are welcome to cross-reference GitHub comments in your review for LiveCoMS.

**Comment by davidlmobley** [comment]
(For example if I, as a reviewer, were already participating in the GitHub discussion on a topic extensively and had, for all practical purposes, already provided my review on GitHub, I would find it considerably simpler to just cross-reference GitHub issues in my review than to type the whole thing again. This won't work in every case, e.g. if anonymity is needed, but my hope is that it will be helpful in some cases.)

**Comment by mrshirts** [comment]
> For the overall questions, I think significance should be discussed. If you don't like yes/no, we could say, "Please comment on the level of significance of this work and its suitability for the journal. Is it likely to have a strong positive impact on a set of readers?"

I'm OK with yes/no in theory.  Usually, for most of the yes/no questions, though, we are interested in WHY they said yes/no.  Different people could have different criteria, and we're more interested in what they say.

> Also, I see a couple of typos:

Thanks, fixed!

> But you aren't going to want to have them all in the review form because that would make it too long/too many boxes; I'd deal with just by linking to the review criteria online and asking them to assess how well it meets the (linked to) criteria for the specific category of document.

Right, I can put that in the form, "Please see [ link place ] for specific questions to address.

> I would also want to point out (again) that they are encouraged to use the GitHub issue tracker to raise any issues they would like to discuss openly with the authors, and reserve the review form for issues which require more confidentiality. . . . Not requiring GitHub, just giving it as an option.

I think as separate thing  is an "instructions for reviewers" which should be a public part of the 'GitHub' site, which should evolve in conjunction with this. And I will discuss the option of providing reviews on GitHub, but that it is optional.

**Comment by davidlmobley** [comment]
Sounds good.

**Comment by dmzuckerman** [comment]
> Not requiring GitHub, just giving it as an option. ...  For example if I, as a reviewer, were already participating in the GitHub discussion on a topic extensively ...
OK @davidlmobley you convinced me.

@mrshirts sorry this is not in the right place but what if we add wording in the instructions like "Reviewers are welcome to reference comments on the github repo, but the review must contain a complete listing of all expected changes in a way that is unambiguous for authors."

**Comment by mrshirts** [comment]
> sorry this is not in the right place
Fixed where it was in the right place.

**Comment by davidlmobley** [comment]
> "Reviewers are welcome to reference comments on the github repo, but the review must contain a complete listing of all expected changes in a way that is unambiguous for authors."

I like this idea.

**Comment by mrshirts** [comment]
OK, see https://github.com/livecomsjournal/livecomsjournal.github.io/pull/143 for the version of the reviewer form provided as part of the reviewer instructions. The ideas here should all be incorporated into those reviewer questions or in other places on the reviewer instruction page.

**Comment by davidlmobley** [comment]
Resolved by the above issue.
